src/main/java/dk/cngroup/lentils/service/StatusService.java

```diff
@@ -94,6 +95,12 @@ private boolean isStatusInDbByCypherAndTeamAndCypherStatus(final Cypher cypher,
         return statusRepository.findByTeam(team);
     }
 
+    public List<Status> getAllByTeamOrderByStage(final Team team) {
+        List<Status> statuses = statusRepository.findByTeam(team);
```

Comment:
Toto zmenit prosim na `statusRepository.findByTeamOrderByCypherStageAsc(team);`. JPA umi nahlednout i dovnitr entit a seradit to jako kdyby podle sloupce prijoinovane tabulky. Musis tuto metodu samozrejme dat i do `StatusRepository`

Reply:
aha, dik, ja prave na to nemohla prijit, jak se dostat dovnitr do te entity a morila se s tim comparatorem - pouceni pro priste, radsi se driv zeptat